telegram/constants.py

```diff
@@ -753,6 +763,25 @@ class MessageType(StringEnum):
     """:obj:`str`: Messages with :attr:`telegram.Message.video_chat_participants_invited`."""
 
 
+class NumberOfUpdatesLimit(IntEnum):
+    """This enum contains limitations for :paramref:`telegram.Bot.get_updates.limit`.
+    The enum members of this enumeration are instances of :class:`int` and can be treated as such.
+
+    .. versionadded:: 20.0
+    """
+
+    __slots__ = ()
+
+    MIN_LIMIT = 1
+    """:obj:`int`: Minimum value allowed for the ``limit`` parameter of
```

Comment:
```suggestion
    """:obj:`int`: Minimum value allowed for the :paramref:`~telegram.Bot.get_updates.limit` parameter of
```
this will actually link to the parameter. Make sure to apply to the rest of the changes as well

Reply:
Yes, I saw examples of this in `constants.py` but it was a bit inconsistent: some of docstrings link to params, some don't.

Reply:
that's because `:paramref:` is a new thing we started in v20, we should use it wherever possible.

Reply:
Got it. Maybe it could be a separate task to add `:paramref:`s to existing docstrings. I'll make sure I add them in the classes I add/change.